### ISSUE TITLE
/things returns empty array

### ISSUE THREAD

**Issue by dljcollette** [creativesoftwarefdn/weaviate#838]
`/things` returns `{ things: [] }`. I was expecting it to return nodes in the array in the `things` object or to return an array with nodes if the `things` object is removed like in all the bodies.

**Comment by etiennedi** [comment]
@dljcollette  Thanks, that's definitely something I must have missed in #826. Just to reiterate, is it only the form that's wrong (i.e. unnecessary "things" field inside the response) or is it also a bug that the array is empty when it shouldn't be?

**Comment by dljcollette** [comment]
@etiennedi it's also empty when it shouldn't be.

**Comment by dljcollette** [comment]
@etiennedi here are the steps to reproduce: 

`POST http://localhost:8080/weaviate/v1/schema/things/` with body `{ class: "Animal", description: "", keywords: [] }` returns `{ class: "Animal", properties: null }`

`POST http://localhost:8080/weaviate/v1/schema/things/Animal/properties/` with body `{ cardinality: "atMostOne", dataType: ["string"], description: "", keywords: [], name: "name" }` returns ` { cardinality: "atMostOne", dataType: ["string"], name: "name" }`

`POST http://localhost:8080/weaviate/v1/things` with body `{ class: "Animal", schema: {name: "Sheep"} }` returns `{ class: "Animal", creationTimeUnix: 1555496336123, id: "1ff39b2d-01b0-4b5c-828d-16039294ffa5", schema: {name: "Sheep"} }`

`GET http://localhost:8080/weaviate/v1/things` returns `{ things: [] }`

**Comment by etiennedi** [comment]
Hi @dljcollette,

unfortunately I still can't reproduce it. Just tried exactly your steps and it works fine for me:

```
$ curl -H 'Content-Type: application/json' -X POST http://localhost:8080/weaviate/v1/schema/things/ -d '{ "class": "Animal", "description": "", "keywords": [] }'
{"class":"Animal","properties":null}

$ curl -H 'Content-Type: application/json' -X POST http://localhost:8080/weaviate/v1/schema/things/Animal/properties/ -d '{ "cardinality": "atMostOne", "dataType": ["string"], "description": "", "keywords": [], "name": "name" }'
{"cardinality":"atMostOne","dataType":["string"],"name":"name"}

$ curl -H 'Content-Type: application/json' -X POST http://localhost:8080/weaviate/v1/things -d '{ "class": "Animal", "schema": {"name": "Sheep"} }'
{"class":"Animal","creationTimeUnix":1555497358846,"id":"9c765553-6cf2-4470-912c-382c0bdcd8bc","schema":{"name":"Sheep"}}

$ curl localhost:8080/weaviate/v1/things
{"things":[{"class":"Animal","creationTimeUnix":1555497358846,"id":"9c765553-6cf2-4470-912c-382c0bdcd8bc","schema":{"name":"Sheep"}}],"totalResults":1}
```

I wouldn't rule out that this was buggy in a previous version, but fixed in the meantime. I just tried with `0.12.71-79efeb0`. Could you verify with exactly this version, too? Thanks

**Comment by dljcollette** [comment]
Seems to be solved in version `0.12.71-79efeb0` with the updated configuration.

**Comment by etiennedi** [comment]
Awesome, thanks for the feedback, closing.
